src/components/cylc/log/Log.vue

```diff
@@ -28,21 +28,49 @@ export default {
   props: {
     placeholder: {
       type: String,
-      default: 'Waiting for logs',
       required: false
     },
+    timestamps: {
+      type: Boolean,
+      required: false,
+      default: true
+    },
     logs: {
       type: Array,
       required: true
     }
   },
+  data () {
+    return {
+      match: ''
+    }
+  },
   computed: {
     computedLogs () {
       if (this.logs.length > 0) {
-        return this.logs
-      } else {
+        if (!this.timestamps) {
+          return this.updateLogs()
+        } else return this.logs
+      } else if (this.placeholder) {
         return [this.placeholder]
+      } else {
+        return []
+      }
+    }
+  },
+  methods: {
+    updateLogs () {
+      return this.logs.map((logLine) => {
+        return this.stripTimestamp(logLine)
+      })
+    },
+    stripTimestamp (logLine) {
```

Comment:
I feel that this ought to be unit-tested - have I missed it being tested somewhere?